app/models/tslr_claim.rb

```diff
@@ -120,6 +120,10 @@ def student_loan_country_with_one_plan?
     StudentLoans::PLAN_1_COUNTRIES.include?(student_loan_country)
   end
 
+  def address_verified?
+    (ADDRESS_ATTRIBUTES & verified_fields).any?
```

Comment:
I like this approach, but I do worry - what if we only get back a postcode from Verify? (and would that even be A Thing?). If it's a possible outcome, maybe we should consider the address unverified?

Reply:
Yeah, I thought about that and decided if we only got back a postcode (which I don't think will happen – Verify should always give us an actual address, right?), then we should count it as "verified" and deal with the consequences then.